covidX/settings.py

```diff
@@ -177,3 +178,8 @@
     "SCHEMA_OUTPUT": "schema.json",
     "SCHEMA_INDENT": 2,
 }
+
+ALGOLIA = {
+            'APPLICATION_ID': 'YourApplicationID',
```

Comment:
Codacy found an issue: [Wrong hanging indentation (remove 8 spaces).](https://app.codacy.com/gh/Xcov19/covidX/pullRequest?prid=6123426)

Reply:
i'm new in open source :octocat:

Reply:
@DataKhukhua please run "Black" to format this file.